Gems/Atom/Feature/Common/Assets/Shaders/Materials/StandardPBR/StandardPBR_Shadowmap_WithPS.shader

```diff
@@ -1,7 +1,10 @@
 {
-    "Source" : "./StandardPBR_Shadowmap_WithPS.azsl",
+    "Source" : "./StandardPBR_DepthPass_WithPS.azsl",
```

Comment:
I think you meant to have StandardPBR_Shadowmap_WithPS.azsl here.

Reply:
Actually no, defining SHADOWMAP in the .shader file eliminates the need to have StandardPBR_Shadowmap_WithPS.azsl, I just forgot to delete the file